debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -102,6 +102,11 @@ public void start(Configuration config) {
                 LOGGER.warn("unable to load info of replication slot, Debezium will try to create the slot");
             }
 
+            if (previousOffset == null && slotInfo != null) {
+                LOGGER.info("No previous offset found");
```

Comment:
II would like to use Debezium in embedded mode with MemoryOffsetBackingStore. I don't have the need for a persisted offset storage between restarts. So in our Kubernetes cluster where restarts are common, this would generate unnecessary WARN messages.

However, I know this is not a common use case. So I will happily change to WARN if you think that is the more appropriate.